src/abstractdataframe/io.jl

```diff
@@ -178,7 +178,7 @@ end
         write(io, "<th>$column_name</th>")
     end
     write(io, "</tr>")
-    tty_rows, tty_cols = Base.displaysize(io)
+    tty_rows, tty_cols = Compat.displaysize()
```

Comment:
Oh, okay. I'm surprised the `io` thing didn't make it into Compat then. I'll revert the most recent commit. Was it okay prior to this commit?